lib/landrush/cap/host/redhat/restart_dnsmasq.rb

```diff
@@ -4,9 +4,9 @@ module Redhat
       module RestartDnsmasq
         class << self
           SED_COMMAND = <<-EOF.gsub(/^ +/, '')
-          sudo sed -i.orig '/nameserver/i\\
-          nameserver 127.0.0.1 # Added by landrush, a vagrant plugin
-
+          sudo sed -i.orig '1 i\
+          # Added by landrush, a vagrant plugin \\
+          nameserver 127.0.0.1 \\
```

Comment:
I kind of like the blank line, since it create a more visual separation as well :-)